data-raw/render_mexico_codes.R

```diff
@@ -1,13 +1,13 @@
 # mexico region codes
+# For ISO-3166-2 codes, see: https://www.iso.org/obp/ui/#iso:code:3166:MX
+# ISO codes were matched by name to INEGI codes in source data
+# Note that Level 1 codes are the first 2 characters of Level 2 codes
+# Level 2 codes remain as provided in source data
 mexico_codes <- vroom::vroom("data-raw/mexico_codes.csv")
 mexico_codes <- tibble::tibble(
-  level = c("level_1_region", "level_2_region"),
-  name = c("iso_3166_2", "inegi_code"),
-  codes = list(
-    dplyr::select(mexico_codes, region_level_1, iso_code) %>%
-      unique(),
-    mexico_codes
-  )
+  level = "level_1_region",
```

Comment:
I thought the decision was to not go with a nested nibble format?

Reply:
this is not nested? this line just adds an extra column to the mexico csv (though i'd prefer to just add it in the csv directly)

Reply:
it has a nibble as a variable?

Reply:
somehow doing this in fact adds on the whole tibble like a left_join, repeating any named variable over the rows of the tibble (joe seems to have found this out, i was also suprised). 

in any case i'm re-writing the mexico_codes csv as it's annoying to have a mix of code and csv when it's not clear where the csv source comes from so can't be recreated anyway.

Reply:
Makes sense. So to be clear the target structure is: 

A table with each row having a level l, a region, and a code where a code is a single identifying character string